src/module/behaviourHelpers.js

```diff
@@ -0,0 +1,10 @@
+export const skipRollDialogCheck = (event) => {
+    const invertedCtrlBehavior = game.settings.get(game.system.id, "invertedCtrlBehavior");
+
+    console.log('Helper triggered.')
```

Comment:
I think we ought to remove this console.log.

Reply:
Haha indeed :)